TestSymbolizationPath failing on Plan 9

Since changeset [f90721db3d](https://github.com/google/pprof/commit/f90721db3d0fcaa8da63a30cc963e2d7d3cbedf7), TestSymbolizationPath is failing on Plan 9.

```
--- FAIL: TestSymbolizationPath (0.02s)
	fetch_test.go:79: :/prod/path/binary:abcde10001, want /tmp/home918705075/pprof/binaries/abcde10001/binary, got /prod/path/binary
FAIL
FAIL	cmd/vendor/github.com/google/pprof/internal/driver	0.484s
```

The test is failing because internal/driver/fetch.go has been changed to handle
the correct home directory environment variable on Plan 9 and Windows,
but internal/driver/fetch.go still uses the $HOME environment variable.

--- a/internal/driver/fetch_test.go
+++ b/internal/driver/fetch_test.go
@@ -38,7 +38,7 @@ func TestSymbolizationPath(t *testing.T) {
 	}
 
 	// Save environment variables to restore after test
-	saveHome := os.Getenv("HOME")
+	saveHome := os.Getenv(homeEnv())
 	savePath := os.Getenv("PPROF_BINARY_PATH")
 
 	tempdir, err := ioutil.TempDir("", "home")
@@ -50,7 +50,7 @@ func TestSymbolizationPath(t *testing.T) {
 	os.Create(filepath.Join(tempdir, "pprof", "binaries", "abcde10001", "binary"))
 
 	obj := testObj{tempdir}
-	os.Setenv("HOME", tempdir)
+	os.Setenv(homeEnv(), tempdir)
 	for _, tc := range []struct {
 		env, file, buildID, want string
 		msgCount                 int
@@ -79,7 +79,7 @@ func TestSymbolizationPath(t *testing.T) {
 			t.Errorf("%s:%s:%s, want %s, got %s", tc.env, tc.file, tc.buildID, tc.want, file)
 		}
 	}
-	os.Setenv("HOME", saveHome)
+	os.Setenv(homeEnv(), saveHome)
 	os.Setenv("PPROF_BINARY_PATH", savePath)
 }
 

